apps/wa/wa-task-batch-reconfiguration/aat/01.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: wa-task-batch-reconfiguration
+spec:
+  values:
+    job:
+      schedule: "2,6,10,14,18,22,26,30,34,38,42,46,50,54,58 * * * *"
+      environment:
+        JOB_NAME: RECONFIGURATION
+    global:
```

I doubt if u will need global flags in there. 
```suggestion
    global:
```

---

removed